src/server/game/Movement/MotionMaster.cpp

```diff
@@ -382,6 +382,7 @@ void MotionMaster::MoveJump(float x, float y, float z, float speedXY, float spee
     init.MoveTo(x, y, z, false);
     init.SetParabolic(max_height, 0);
     init.SetVelocity(speedXY);
+    init.SetFacing(o);
```

Comment:
Not all jump movesplines have a facing attached. These cases should be handled.

How about passing `nanf` or similar value as orientation in those cases, and not setting facing for these?